### DISCUSSION TITLE
New space, who dis

### DISCUSSION THREAD

**Discussion by Hanse00** [super-linter/super-linter#4500] [Maintainers]
In an effort to be more transparent with the community, @zkoppert and I decided on our call today to create this discussion forum. 

The idea is that we'll use it to discuss the ongoing business of maintaining the project, rather than using Slack as we have done historically.

**Comment by Hanse00** [comment]
For the information of anyone who is curious: We've had other maintainers in the past, currently it is primarily @zkoppert and myself dedicating time to this project. Since Zack and I are both local to the Portland metro area, we've gotten together physically on a couple of occasions to discuss the future of the project, and also meet semi-regularly via video conferencing.

However we want to be good stewards of the project, and allow other members of the community to have input, and understand how we work. Therefore we've decided to attempt to use discussions as our primary discussion forum.

**Reply by ferrarimarco** [reply]
and @ferrarimarco :)
